src/libs/ReportUtils.ts

```diff
@@ -4271,7 +4291,9 @@ function goBackToDetailsPage(report: OnyxEntry<Report>, backTo?: string) {
         return;
     }
 
-    Navigation.goBack(ROUTES.REPORT_SETTINGS.getRoute(report?.reportID ?? '-1', backTo));
+    if (report?.reportID) {
+        Navigation.goBack(ROUTES.REPORT_SETTINGS.getRoute(report.reportID, backTo));
+    }
```

Comment:
NAB: Let's also log a warning here if there's no reportID for some reason. We could also navigate back to the concierge chat in that case which would be a better fallback than doing nothing.